lib/rules/no-useless-constructor.js

```diff
@@ -25,13 +25,20 @@ module.exports = function(context) {
             return false;
         }
 
-
         return body[0].expression.arguments.every(function(arg, index) {
-            return (arg.type === "Identifier" && arg.name === ctorParams[index].name) ||
```

Comment:
Nice refactoring! I can create a PR but I don't wanna take credit for the work you did. Would it be hard for you to create a PR for this? If not I'll do it.